src/game/gameTimer.js

```diff
@@ -1,11 +1,8 @@
-import { finishGame } from "./gameResult";
-
 // container class where will be displayed timer. Change classname if it is necessary
-const TIMER = document.querySelector(".timer");
-
 let gameTime;
 
-export function setTimer(time = 10) {
+function setTimer(time = 10, cb) {
+  const TIMER = document.querySelector(".timer");
```

Comment:
`SCREAMING_SNAKE_CASE` is used to make constants more verbose and visible for a reader. If you are creating const in a function it doesn't have then any real value and it can be called just a `timer`.

Reply:
changed